src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressFilterElement.java

```diff
@@ -136,46 +124,25 @@ public SuppressFilterElement(String files, String checks,
      */
     public SuppressFilterElement(Pattern files, Pattern checks, Pattern message, String moduleId,
             String lines, String columns) {
-        if (files == null) {
-            filePattern = null;
-            fileRegexp = null;
-        }
-        else {
-            filePattern = files.pattern();
-            fileRegexp = files;
-        }
-        if (checks == null) {
-            checkPattern = null;
-            checkRegexp = null;
-        }
-        else {
-            checkPattern = checks.pattern();
-            checkRegexp = checks;
-        }
-        if (message == null) {
-            messagePattern = null;
-            messageRegexp = null;
-        }
-        else {
-            messagePattern = message.pattern();
-            messageRegexp = message;
-        }
+        fileRegexp = files;
+        checkRegexp = checks;
+        messageRegexp = message;
         this.moduleId = moduleId;
         if (lines == null) {
             linesCsv = null;
             lineFilter = null;
         }
         else {
             linesCsv = lines;
-            lineFilter = new CsvFilterElement(lines);
+            lineFilter = new CsvFilterElement(linesCsv);
```

Comment:
Why can't we leave this as it was before? It is logically equivalent.
Why don't we remove `linesCsv` if it is always the assignment of `lines`?

Reply:
restored.

Reply:
we use it to store a value, may be we can take it from CsvFilterElement but it is abit more refactoring with no much value.